assets/templates/index.gohtml

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./assets/static/css/style.css">
```

Comment:
There's a bunch of meta we need here, namely:

- robots noindex,nofollow,noarchive
- description
- keywords
- any og data
- favicons (chrome will complain otherwise)
- apple and android styles:

    ```html
    <meta name="apple-mobile-web-app-capable" property="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" property="apple-mobile-web-app-status-bar-style" media="(prefers-color-scheme: light)" content="white">
    <meta name="apple-mobile-web-app-status-bar-style" property="apple-mobile-web-app-status-bar-style" media="(prefers-color-scheme: dark)" content="black">
    <meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
    <meta name="theme-color" media="(prefers-color-scheme: dark)" content="black">
    ```
- csrf injection

Reply:
While not a huge deal, some of this might be overkill for an internal app? Mainly, the search meta stuff (description, keywords, og)

Regardless, we should move this part into a separate template like head.html or something and import it then.

Reply:
keywords - is this more applicable to searching/indexing pages?
any og data - i can add title, description. Not sure if we image, if so is there a particular image you had in mind?
favicons (chrome will complain otherwise) - Is there a base image we are using, perhaps Magneto did something here.
csrf injection - unfamiliar but Ill look into this

Reply:
Fair on keywords. Description and OG data will be pulled when links are posted in chat/slack, so we should add it.

Reply:
I just sent you a possible favicon in chat.